index.js

```diff
@@ -39,6 +86,30 @@ app.get("/messages", (req, res) => {
   res.json(messages.slice(afterId));
 });
 
+app.put("/message/:id", (req, res) => {
+  if (!messages[req.params.id - 1]) {
+    return res.status(404).json({ error: "Unknown item" });
+  }
+  if (req.body.message) {
+    messages[req.params.id - 1].message = req.body.message;
```

Comment:
Now that we can deleting things we can't update the messages like this anymore. We need to find the index of the message with message.id === afterId and then update based on that index

Reply:
think this should now work off of index